packages/react-sdk/src/components/Layout/useFullscreenMode.ts

```diff
@@ -39,8 +39,10 @@ export function useFullscreenMode(): UseFullScreenModeResult {
   );
 
   const setFullscreenMode = useCallback((value: boolean) => {
+    const container =
```

Comment:
Even it is cheap, `container` is only used inside the `if`-branch. Could be moved to inside of it.